lib/susedistribution.pm

```diff
@@ -160,6 +160,9 @@ sub init_cmd {
         $testapi::cmd{sync_without_daemon} = "alt-s";
     }
     ## keyboard cmd vars end
+    if (check_var('FLAVOR', 'Online')) {
+        $testapi::cmd{expertpartitioner} = "alt-x";
```

Comment:
$testapi::cmd{expertpartitioner} -> $testapi::cmd{expert_partitioner}, just a suggestion , not sure, i think  the change should follow united format？

Reply:
It's a global testapi defined in lib/susedistribution.pm, all others tests follow this.

Reply:
please refer to this pr   .  https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10603
shortcut key are dynamic generated  .

Reply:
@jerrytt Thanks, updated as your PR